CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Node.js Agent for Google Cloud Debug ChangeLog
 
+## 2017-03-14, Version 1.0.0 (Experimental), @dominicdkramer
+
+### Semver-major changes
+
+* Evaluation of expressions is no longer allowed by default.  It can be enabled using the `allowExpressions` configuration option.
+* When the agent is used with a Google Cloud Functions function, the function name is used as part of the debugger's description.
```

Comment:
I thought all of these could possibly break previous code, and so I added them just to be safe and to get a second look.  I have now removed them except for the change to the evaluation of expressions.